Dockerfile

```diff
@@ -10,9 +9,9 @@ RUN make release
 RUN mv euterpe /tmp/euterpe
 RUN /tmp/euterpe -config-gen && sed -i 's/localhost:9996/0.0.0.0:9996/' /root/.euterpe/config.json
 
-FROM debian:buster
+FROM alpine:latest
 
-RUN apt-get update && apt-get install -y libtagc0 libicu63
+RUN apk add --update taglib-dev libc-dev icu-dev
```

Comment:
Aren't there non "-dev" versions of the packages? If there are I expect the image to become even smaller. I will give it a try.

Reply:
Strangely enough I am getting "not permitted" during building which is a complete surprise for me:

```
Step 5/15 : RUN make release
 ---> Running in 4e4bb060e991
go build \
	--tags "sqlite_icu" \
	-ldflags "-X github.com/ironsmile/euterpe/src/version.Version=`git describe --tags --always`" \
	-o euterpe
make: /bin/sh: Operation not permitted
make: *** [Makefile:9: release] Error 127
```

Reply:
Apparently the root problem is [this](https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2). Although I don't know what to abut it :thinking: I want to be able to build the image locally on my machine.

Using the Golang 1.16-alpine-3.13 image seems to do the job.

Reply:
Yeah, I've had intermittent issues with `alpine3.14` too, so I'd be happy to switch it over to that.

There doesn't appear to be a non-dev package for `libc`, but the others do, so I'm happy to switch to those for this second `apk add` in the file.